data/json/mapgen_palettes/roof_palette.json

```diff
@@ -56,9 +56,18 @@
     "liquids": { "Я": { "liquid": "water_clean", "amount": [ 0, 300 ] } },
     "nested": {
       "1": {
-        "chunks": [ [ "residential_1x2_solar", 6 ], [ "residential_2x2_solar", 2 ], [ "residential_1x3_solar", 3 ], [ "null", 88 ] ]
+        "//": "11% chance of solar, 20% chance it's v2",
+        "chunks": [
+          [ "residential_1x2_solar", 24 ],
+          [ "residential_2x2_solar", 8 ],
+          [ "residential_1x3_solar", 12 ],
+          [ "residential_1x2_solar_v2", 6 ],
+          [ "residential_2x2_solar_v2", 2 ],
+          [ "residential_1x3_solar_v2", 3 ],
+          [ "null", 440 ]
+        ]
       },
-      "2": { "chunks": [ "industrial_1x1_solar" ] },
+      "2": { "chunks": [ [ "industrial_1x1_solar", 4 ], [ "industrial_1x1_solar_v2", 1 ] ] },
```

Comment:
Is there a way to set that as a param or switch so that on any given roof map you will only have v1 or v2 solar panels?   Or is the difference just recognizing that some panels were installed at different dates?

Reply:
The difference was not intentional, but it's doesn't appear that there's a way to use parameters for chunk selection, or another way to ensure that the solar panel type is consistent.